lib/langchain/conversation.rb

```diff
@@ -17,7 +17,7 @@ module Langchain
   #     end
   #
   class Conversation
-    attr_reader :options
+    attr_reader :options, :memory
```

Comment:
WIth this, it will open possibilities to manipulate the memory as the dev want

Reply:
I'm not sure about this change. Even if we wanted to do this, I think it could be discussed separately and doesn't need to be part of this PR. Or am I missing something?

Reply:
It is convenient when someone (like me) wants to manipulate memory for various purposes.